srcpkgs/msmtp/template

```diff
@@ -3,10 +3,10 @@ pkgname=msmtp
 version=1.8.2
 revision=1
 build_style=gnu-configure
-configure_args="--with-tls=openssl $(vopt_with idn libidn)
+configure_args="$(vopt_with msmtpd) $(vopt_with tls openssl) $(vopt_with idn libidn)
  $(vopt_with sasl libgsasl) $(vopt_with gnome libsecret)"
 hostmakedepends="pkg-config"
-makedepends="libressl-devel $(vopt_if idn libidn-devel)
+makedepends="$(vopt_if openssl libressl-devel gnutls-devel) $(vopt_if idn libidn2-devel)
```

Comment:
don't create a openssl build options, just use gnutls-devel.

I don't want to give an easy way for people to use stuff dis-encouraged by upstream, specially when there is a recommended alternative.

Reply:
Got you. Just updated the PR.